CHANGELOG.md

```diff
@@ -388,6 +394,36 @@ Other minor additions
   _⇔_ : REL A B ℓ₁ → REL A B ℓ₂ → Set _
   ```
 
+* Added new functions to `Data.Fin.Base`:
+  ```agda
+  group : ∀ n k → Fin (n ℕ.* k) → Fin k × Fin n
```

Comment:
I don't think this is a good name. `factor` maybe? It is part of an important equivalence between the two given types, so it is rather important to give it an expected name.

Reply:
`quotRem`?

Reply:
I renamed it to `quotRem` since Haskell has a similarly named function. I also made `n` implicit.